projects/spring-boot/ConfigurationMetadataRepositoryJsonBuilderFuzzer.java

```diff
@@ -0,0 +1,12 @@
+import org.springframework.boot.configurationmetadata.ConfigurationMetadataRepositoryJsonBuilder;
```

Comment:
Can you add a license header please?

Reply:
I added a license header and rebased on master